src/pages/detail/tabs/members/GroupMembersTab.tsx

```
@@ -1,7 +1,25 @@
+import { getUserInfo } from "@/apis/auth";
+import { UserInfo } from "os";
+import useSWR from "swr";
+
 interface GroupMembersTabProps {}
 
 const GroupMembersTab = ({}: GroupMembersTabProps) => {
```

Biome

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)